libs/utils/trace_analysis.py

```diff
@@ -386,7 +387,9 @@ def plotTasks(self, tasks=None, signals=None):
             residencies: enable the generation of the CPUs residencies plot
 
         Args:
-            tasks   (list): the list of tasks to plot
+            tasks   (list): the list of task names and/or PIDs to plot.
+                            Numerical PIDs and string task names can be mixed
+                            in the same list.
                             default: all tasks defined at TraceAnalysis
                             creation time are plotted
             signals (list): list of signals (and thus plots) to generate
```

Comment:
We should decide what documentation style to use. I've always used *sphinxy* style so far, because it's used in TRAPpy and BART